include/erlang_ls.hrl

```diff
@@ -545,6 +545,22 @@
                                   , active_parameters => number()
                                   }.
 
+%%------------------------------------------------------------------------------
+%% Formatting
+%%------------------------------------------------------------------------------
+-type formatting_options() :: #{ tabSize           := integer()
+                               , insertSpaces      := boolean()
+                               %% Spec says further properties must
```

Comment:
Could you convert this comment into actual type specs for the `furtherProperties`?

Reply:
I am not sure how it would be represented, to be honest.   I guess a list of (key,value) pairs, would sort of capture it.

I understand the intent to be that you can put additional formatting options into the structure, that are first-class fields.  i.e. they are not wrapped in any additional structure.

Reply:
Read the specifications myself. I would say we should remove the `furtherProperties` field altogether here, and add individual properties once handled.

Reply:
Agree, and leave the comment.